lib/fs.js

```diff
@@ -783,6 +783,10 @@ fs.renameSync = function(oldPath, newPath) {
 
 fs.truncate = function(path, len, callback) {
   if (typeof path === 'number') {
+    process.emitWarning(
+      'Using fs.truncate with file descriptor deprecated. In the future, ' +
+      'use fs.ftruncate with file descriptor',
```

Comment:
Nit - this should be indented with two more spaces while landing. The same applies to the other deprecation messages.

Reply:
The message itself could also be improved by not saying `In the future ...` but just `Please use ... instead`.

Reply:
I agree with switching to `Please use ...`. Additionally, the occurrences of "with file descriptor" should likely be "with a file descriptor"